code/datums/outfits/outfit_admin.dm

```diff
@@ -115,8 +122,8 @@
 	name = "Syndicate Strike Team"
 
 /datum/outfit/admin/syndicate_strike_team/equip(mob/living/carbon/human/H, visualsOnly = FALSE)
-	return H.equip_syndicate_commando(FALSE, TRUE)
-
+	. = H.equip_syndicate_commando(FALSE, TRUE)
+	H.faction += "syndicate"
```

Comment:
```suggestion
	H.faction |= "syndicate"
```

Reply:
This can be comitted into your branch with no changes tourte